Cineaste/Custom UI/View.swift

```diff
@@ -189,4 +184,35 @@ class VoteView: View {
             .setContentCompressionResistancePriority(.required,
                                                      for: .vertical)
     }
+
+    @objc
+    func setBorderWidth() {
+        let borderWidth: CGFloat
+
+        if #available(iOS 11.0, *) {
+            borderWidth = UIFontMetrics.default.scaledValue(for: 1)
```

Comment:
This works so good... 😍 Glad, I learned about this in here https://developer.apple.com/videos/play/wwdc2017/245